packages/ember-metal/lib/deprecate_property.js

```diff
@@ -23,7 +23,7 @@ import { set } from "ember-metal/property_set";
 
 export function deprecateProperty(object, deprecatedKey, newKey) {
   function deprecate() {
-    Ember.deprecate('Usage of `' + deprecatedKey + '` is deprecated, use `' + newKey + '` instead.');
+    Ember.deprecate(`Usage of "${deprecatedKey}" is deprecated, use "${newKey}" instead.`);
```

Comment:
The consistency on doing this throughout the code isn't that great right now, but worth not slipping at least :)